File: linkdisk_flutter/副本lib/model/TransferFileItem.dart
import 'dart:io';

class TransferFileItem {
  // 私有属性（对应 Java private）
  final File _sourceFile;
  final String _relativePath;
  final int _size;

  // 构造函数
  TransferFileItem(this._sourceFile, this._relativePath)
      : _size = _sourceFile.lengthSync();

  // getter 方法（对应 Java getXxx）
  File get sourceFile => _sourceFile;
  String get relativePath => _relativePath;
  int get size => _size;
}

File: linkdisk_flutter/副本lib/model/TransferFileItem.java
package LinkDisk.model;

import java.io.File;

public class TransferFileItem {

    private File sourceFile;

    private String relativePath;

    private long size;

    public TransferFileItem(File sourceFile, String relativePath) {
        this.sourceFile = sourceFile;
        this.relativePath = relativePath;
        this.size = sourceFile.length();
    }

    public File getSourceFile() {
        return sourceFile;
    }

    public String getRelativePath() {
        return relativePath;
    }

    public long getSize() {
        return size;
    }
}